frontend/sentry.properties

```
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
+defaults.org=ifixit
+defaults.project=react-commerce
+cli.executable=../../../../.npm/_npx/a8388072043b4cbc/node_modules/@sentry/cli/bin/sentry-cli
```

This seems like it won't be stable. That looks like the kind of path that's gonna change over time. Is there a default that's reasonable if we just don't set it?

---

Good point. I think if we omit this field, Sentry will look for a globally installed `sentry-cli`. `../node_modules/@sentry/cli/bin/sentry-cli` will work, though I'm not sure it's much better.

```suggestion
cli.executable=../node_modules/@sentry/cli/bin/sentry-cli
```

---

https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#configure-sentry-cli

---

I did this in https://github.com/iFixit/react-commerce/pull/321/commits/cfbe26e5f724f1af9ec0e88e64a7a1ae3b3e0558, lmk if you have other ideas